frontend/src/components/SimpleBar/SimpleBar.js

```diff
@@ -7,26 +8,60 @@ import { NumberFormatType } from 'utils/propTypes'
 import FieldNumber from 'components/FieldNumber'
 
 const SimpleBar = ({ value, percentage, numberFormat }) => {
+  const containerRef = useRef(null)
+  const fieldRef = useRef(null)
+  const [fullWidth, setFullWidth] = useState(containerRef.current?.clientWidth)
+  const fieldWidth = fieldRef.current?.clientWidth
+  const handleContainerResize = useCallback(() => setFullWidth(containerRef.current?.clientWidth), [setFullWidth])
+  const getContainerNode = useCallback(
+    node => {
+      if (node) {
+        containerRef.current = node
+        handleContainerResize()
+      }
+    },
+    [handleContainerResize],
+  )
+  const getFieldNode = useCallback(node => {
+    if (node) {
+      fieldRef.current = node
+    }
+  }, [])
+
+  useEffect(() => {
+    window.addEventListener('resize', handleContainerResize)
+    return () => window.removeEventListener('resize', handleContainerResize)
+  }, [handleContainerResize])
+
+  const width = useMemo(
+    () => Math.max((fullWidth - fieldWidth - 8) * percentage, 8),
+    [fullWidth, fieldWidth, percentage],
+  )
+
   return (
-    <Grid container spacing={1} alignItems="center" flexWrap="nowrap">
-      <Grid item flex={1}>
-        <Box sx={{ position: 'relative', width: `${(percentage || value) * 100}%` }}>
```

Comment:
@leo-esaki What are the benefits to the changes in this branch versus simply putting `minWidth: '8px'` here?